study-frontend/src/Components/LoginBox.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect} from 'react';
+import { Button, Checkbox, Form, Input } from 'antd';
+import { Authentication } from '../Services/Authentication/Authentication';
+import { useNavigate } from 'react-router-dom';
+import { routes } from '../constants';
+import '../cssFiles/loginSignInPage.css'
+
+const onFinishFailed = (errorInfo: any) => {
+  console.log('Failed:', errorInfo);
+};
+
+
+
+type Props = {
+  handleClick : ()=> void
+
+};
+
+const Login = ({handleClick} : Props ) => 
+{    
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState(""); 
+  const [isError, setIsError] = useState(false);
+  const navigate = useNavigate(); 
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+
+  
+  const handleSubmit = async () => {
+    if(email !== "" && password !== "")
+    {
+      try {
+        const response = await Authentication({ email, password });
+        
+        
+        sessionStorage.setItem("jwt", response.token);
```

Comment:
Remove this logic from this component add it to a figher level so that you can access it from any other child component. What we talked in the office.

Reply:
Да това със запазването на sessionStorage ще го направя със zustand, но първо трябва да мине това и да го мъеджна с мастера за да мога да работя по другата част.